packages/react/src/components/GroupedList/GroupedListV2.base.tsx

```diff
@@ -396,14 +406,27 @@ export const GroupedListV2FC: React.FC<IGroupedListV2Props> = props => {
   const renderHeader = (item: IHeaderGroupedItem, flattenedIndex: number): React.ReactNode => {
     const group = item.group;
 
+    let ariaProps;
+    if (role === 'treegrid') {
+      // GroupedList default role
+      ariaProps = {
+        ariaLevel: group.level ? group.level + 1 : 1,
+        ariaSetSize: groups ? groups.length : undefined,
+        ariaPosInSet: item.groupIndex,
+      };
+    } else {
+      // Grouped DetailsList
+      ariaProps = {
+        ariaRowIndex: flattenedIndex,
```

Comment:
I didn't verify in the examples, but does `flattenedIndex` already take into account whether there are column headers?

Reply:
It does. [This is handled internally by DetailsList](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/DetailsList/DetailsList.base.tsx#L404).

![screenshot of devtools showing the correct aria-rowindex value](https://user-images.githubusercontent.com/93940821/201445491-2b7890de-0930-4f0d-840f-62242a4ddb63.png)

Reply:
In verifying this I did notice that the column headers do not have an `aria-rowindex` value set. Do they need one?

Reply:
not really -- they're always present in the DOM and will always be the first row. `aria-rowindex` is auto-calculated based on DOM if it's not explicitly defined.